android/build_boinc_arm.sh

```diff
@@ -23,14 +23,26 @@ export TOOLCHAINROOT="$NDK_ROOT/toolchains/llvm/prebuilt/linux-x86_64/"
 export TCBINARIES="$TOOLCHAINROOT/bin"
 export TCINCLUDES="$ANDROIDTC/arm-linux-androideabi"
 export TCSYSROOT="$TOOLCHAINROOT/sysroot"
+export VCPKG_DIR=$VCPKG_ROOT/installed/arm-android
+
+VCPKG_CONFIG_FLAG=""
+VCPKG_LDFLAG=""
+
+if [ $BUILD_WITH_VCPKG = "yes" ]; then
+    VCPKG_LDFLAG="-L$VCPKG_DIR/lib"
+    VCPKG_CONFIG_FLAG="--with-libcurl=$VCPKG_DIR --with-ssl=$VCPKG_DIR --enable-vcpkg"
+else
+    VCPKG_CONFIG_FLAG="--with-ssl=$TCINCLUDES"
```

Comment:
You use variable with `VCPKG_` prefix for builds without vcpkg, it's misleading, and should be changed.
It's the same for other similar scripts, please change them also, I don't want to publish the same comment in any file

Reply:
fix